source/components/atoms/Select/Select.tsx

```diff
@@ -43,59 +43,74 @@ interface Props {
   style?: ViewStyle;
   onBlur: (value: string) => void;
   onOpen: (event: undefined, isSelect: true) => void;
+  onClose: (event: undefined, isSelect: true) => void;
```

Comment:
```js
onClose: (event: undefined, isSelect: true) => void;
```

should be:
```js
onClose: (event: undefined, isSelect: boolean) => void;
```

and why are we even using event as parameter when its typed as undefined? I guess that the event should have a type right?

Reply:
The function signature is based on a native event. For `onOpen` for example, it's actual type _should_ be `NativeSyntheticEvent<TextInputFocusEventData>` but strictly speaking doesn't have to be...

My suggestion would be to change it to `event: unknown` instead. `undefined` is incorrect as `event` _is_ defined in most cases.